### ISSUE TITLE
limit should be typed: number

### ISSUE THREAD

**Issue by jacobrosenthal** [jacobrosenthal/node-cryptokitties-cli#11]

**Comment by Deserae** [comment]
As in limit is not optional?

**Comment by jacobrosenthal** [comment]
No.. its currently a string, it needs to have its 'type' set to number so
parseInt() called behind the scenes

see: type:'number' under this link:
https://github.com/yargs/yargs/blob/master/docs/api.md#usagemessagecommand-desc-builder-handler
